steps/xcodegen/0.0.2/step.yml

```diff
@@ -0,0 +1,66 @@
+title: XcodeGen
+summary: |
+  This Bitrise step runs XcodeGen to generate a project.
+description: |
+  This Bitrise step runs XcodeGen to generate a project.
+website: https://github.com/brandtdaniels/xcodegen-bitrise
+source_code_url: https://github.com/brandtdaniels/xcodegen-bitrise
+support_url: https://github.com/brandtdaniels/xcodegen-bitrise/issues
+published_at: 2019-10-15T15:34:57.183896-07:00
+source:
+  git: https://github.com/brandtdaniels/xcodegen-bitrise.git
+  commit: 3457781eb072e5673ddfa3a35806edc21eda8236
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
```

Comment:
XcodeGen currently doesn’t run on Linux

Reply:
Looks like it's currently unused so it shouldn't be an issue

https://devcenter.bitrise.io/jp/bitrise-cli/step-properties/

> host_os_tags : supported operating systems. Currently unused, reserved for future use.